src/Dscp/Util.hs

```diff
@@ -32,3 +42,23 @@ wrapRethrowIO
     :: (Exception e1, Exception e2, MonadCatch m, MonadIO m)
     => (e1 -> e2) -> IO a -> m a
 wrapRethrowIO wrap action = wrapRethrow wrap (liftIO action)
+
+-----------------------------------------------------------
+-- Bytestrings formatting
+-----------------------------------------------------------
+
+toBase :: ByteArrayAccess ba => Base -> ba -> Text
+toBase base = decodeUtf8 @Text @ByteString . convertToBase base
+
+fromBase :: forall ba. FromByteArray ba => Base -> Text -> Either String ba
+fromBase base =
+    convertFromBase base . encodeUtf8 @Text @ByteString >=>
+    fromByteArray @ba @ByteString
+
+toBase64, toHex :: ByteArrayAccess ba => ba -> Text
```

Comment:
We also have it in `serokell-util`. Not saying you necessarily should use it, but...

Reply:
Fair enough, but these ones work for every `ByteArrayAccess`, so I'll stick with them.